graphql-api-gateway-poc/package.json

```diff
@@ -5,7 +5,6 @@
   "main": "index.js",
   "scripts": {
     "start-gateway": "nodemon src/index.js",
```

Comment:
This is something I didn't notice earlier, but -- why are you using nodemon here? The Dockerfile is already set up to run pm2 on `src/index.js`. Is there some advantage to using a custom script with nodemon?

Reply:
This script is not being used for the dockerfile. It was while testing locally. I'll remove it in the next commit. While running the containers we use pm2 only.

Reply:
Ah, makes sense.